tests/test_tcp_controller.py

```diff
@@ -1,342 +1,343 @@
-            await self.hexController.setSoftLimitStatus(X=xb, Y=yb, Z=zb, U=ub, V=vb, W=wb)
-            x, y, z, u, v, w = await self.hexController.getSoftLimitStatus()
-            self.assertEqual(xb, x)
-            self.assertEqual(yb, y)
-            self.assertEqual(zb, z)
-            self.assertEqual(ub, u)
-            self.assertEqual(vb, v)
-            self.assertEqual(wb, w)
-
-            xb, yb, zb, ub, vb, wb = True, True, True, True, True, True
-            await self.hexController.setSoftLimitStatus(X=xb, Y=yb, Z=zb, U=ub, V=vb, W=wb)
-            x, y, z, u, v, w = await self.hexController.getSoftLimitStatus()
-            self.assertEqual(xb, x)
-            self.assertEqual(yb, y)
-            self.assertEqual(zb, z)
-            self.assertEqual(ub, u)
-            self.assertEqual(vb, v)
-            self.assertEqual(wb, w)
-        asyncio.get_event_loop().run_until_complete(doit())
-
-    @unittest.skip("Don't run...")
-    def testSpeedSet(self):
-        """Test set sft limits status."""
-        async def doit():
-            systemVelocity = 1
-            await self.hexController.setSystemVelocity(systemVelocity=systemVelocity)
-            velocity = await self.hexController.getSystemVelocity()
-            self.assertEqual(systemVelocity, velocity)
-
-            systemVelocity = 3
-            await self.hexController.setSystemVelocity(systemVelocity=systemVelocity)
-            velocity = await self.hexController.getSystemVelocity()
-            self.assertEqual(systemVelocity, velocity)
-
-        asyncio.get_event_loop().run_until_complete(doit())
-
-    @unittest.skip("Don't run...")
-    def testVirtualMove(self):
-        """Test virtual move."""
-        async def doit():
-            valid = await self.hexController.validPosition(X=50)
-            self.assertEqual(valid, False)
-
-            valid = await self.hexController.validPosition(X=5)
-            self.assertEqual(valid, True)
-
-            valid = await self.hexController.validPosition(X=5, Y=1, Z=1)
-            self.assertEqual(valid, True)
-
-            valid = await self.hexController.validPosition(X=5, Y=1, Z=1, U=35)
-            self.assertEqual(valid, False)
-        asyncio.get_event_loop().run_until_complete(doit())
-
-
-def similarTo(number1, number2, threshold=0.001):
-    return abs(number1 - number2) < threshold
-
-
-async def inPosition(hexController, Xcmd, Ycmd, Zcmd, Ucmd, Vcmd, Wcmd):
-    Xtgt, Ytgt, Ztgt, Utgt, Vtgt, Wtgt = await hexController.getRealPositions()
-    inPositionValue = (similarTo(Xcmd, Xtgt) and similarTo(Ycmd, Ytgt) and similarTo(Zcmd, Ztgt) and
-                       similarTo(Ucmd, Utgt) and similarTo(Vcmd, Vtgt) and similarTo(Wcmd, Wtgt))
-    print("inPositionValue" + str(inPositionValue))
-    return inPositionValue
-
-
-if __name__ == '__main__':
-    atHexTests = unittest.main()
+"""Tests the TCP controller."""
+# from lsst.ts.ATHexapod.ATHexapodController import ATHexapodController
```

Comment:
Instead of commenting all the code, if you plan on keeping this, try skipping the test with a decorator like 

```
@unittest.skip('Some reason...')
```